### PR TITLE
This fixes multi challenge for u2f

### PR DESCRIPTION
**Pull request by Mipronimo**
If a user has multiple u2f tokens. Only one can be used. This will fix it.
Thanks @jh23453 for the PR #22.
I had to write it down again, because we changed to much in the files and it had to be fixed in auth source and auth proc.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50%23discussion_r248618867%22%2C%20%22https%3A//github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50%23discussion_r248627570%22%2C%20%22https%3A//github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50%23discussion_r248627665%22%2C%20%22https%3A//github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50%23discussion_r248628794%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%203aa6ac1a65ae112166a1643b64eb5074b10515fd%20www/loginform.php%2015%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50%23discussion_r248618867%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20we%20do%20i18n%20here%3F%22%2C%20%22created_at%22%3A%20%222019-01-17T10%3A40%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/1908620%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cornelinux%22%7D%7D%2C%20%7B%22body%22%3A%20%22Fixed%20here%20https%3A//github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50/commits/8050b183489e4d82439288f375fb7efc02b2775c%22%2C%20%22created_at%22%3A%20%222019-01-17T11%3A07%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/35998229%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Mipronimo%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222019-01-17T11%3A07%3A38Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/35998229%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/Mipronimo%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%20%22%2C%20%22created_at%22%3A%20%222019-01-17T11%3A11%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/1908620%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cornelinux%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20www/loginform.php%3AL109-124%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 3aa6ac1a65ae112166a1643b64eb5074b10515fd www/loginform.php 15'></a> &nbsp;<b><a href='https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50#discussion_r248618867'>File: www/loginform.php:L109-124</a></b>
- <a href='https://github.com/cornelinux'></a> Can we do i18n here?
- <a href='https://github.com/Mipronimo'></a> Fixed here https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50/commits/8050b183489e4d82439288f375fb7efc02b2775c

<a href='https://www.codereviewhub.com/privacyidea/simplesamlphp-module-privacyidea/pull/50?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/privacyidea/simplesamlphp-module-privacyidea/pull/50?approve=1'></a>&nbsp;<a href='https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by cornelinux** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50'>Refresh</a>.<a href='#crh-update-none'></a>

File: www/loginform.php [hunk 1]
```diff
@@ -109,11 +109,15 @@
             /* In case of challenge response we do not change the username */
             $state['forcedUsername'] = $username;
             $transaction_id = $errorParams[1];
-            $message = $errorParams[2];
-            $attributes = $errorParams[3];
+            $message = 'Please confirm with one of these tokens: ';
```
Comment (cornelinux) [inline www/loginform.php]: Can we do i18n here?

Comment (Mipronimo) [reply www/loginform.php]: Fixed here https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/50/commits/8050b183489e4d82439288f375fb7efc02b2775c

Comment (Mipronimo) [inline www/loginform.php]: :+1:

Comment (cornelinux) [reply www/loginform.php]: :+1:

**Pull request merged**